src/frame.rs

```diff
@@ -771,16 +779,16 @@ impl Frame<marker::Points> {
     }
 
     /// Gets texture coordinates of each point of point cloud.
-    pub fn texture_coordinates<'a>(&'a self) -> RsResult<&'a [realsense_sys::rs2_pixel]> {
+    pub fn texture_coordinates<'a>(&'a self) -> RsResult<&'a [texture_coordinate]> {
         unsafe {
             let n_points = self.points_count()?;
             let mut checker = ErrorChecker::new();
             let ptr = realsense_sys::rs2_get_frame_texture_coordinates(
                 self.ptr.as_ptr(),
                 checker.inner_mut_ptr(),
-            );
+            ) as *const texture_coordinate;
```

Comment:
Please give a reference that the casting is safe, or use `safe_transmute` instead.

Reply:
Added a safety comment and used `safe_transmute::transmute_many()` to convert from `*const rs2_pixel` to `&[TextureCoordinate]`.